src/utils.rs

```diff
@@ -97,15 +97,24 @@ pub(crate) fn display_string_discreetly(
 /// [1]: https://ethereum.org/en/developers/docs/data-structures-and-encoding/web3-secret-storage.
 pub(crate) fn write_wallet_from_mnemonic_and_password(
     wallet_path: &Path,
+    force_write: bool,
     mnemonic: &str,
     password: &str,
 ) -> Result<()> {
-    // Ensure we're not overwriting an existing wallet or other file.
     if wallet_path.exists() {
-        bail!(
-            "File or directory already exists at {wallet_path:?}. \
-            Remove the existing file, or provide a different path."
-        );
+        if !force_write {
+            println!(
+                "There is an existing wallet at {wallet_path:?}. \
+                Do you wish to replace it with a new wallet? (y/N) "
+            );
+            let mut need_replace = String::new();
+            stdin().read_line(&mut need_replace)?;
+            let need_replace_real = need_replace.trim();
+            if need_replace_real != "Y" && need_replace_real != "y" {
+                return Ok(())
```

Comment:
The error could be "Failed to create a new wallet at "/tmp/.wallet" because a wallet already exists at that location." with `println_red_err`

I think an error is appropriate here because the initial command failed.